CoinbasePro/WebSocket/WebSocket.cs

```diff
@@ -92,6 +92,12 @@ public class WebSocket : IWebSocket
 
         public void Stop()
         {
+            if (webSocketFeed == null)
```

Comment:
hey @chrisw000. just trying to understand the case when this happens. does this only occur if the user is calling stop before start is called?

Reply:
Correct. Obviously its possible to handle it outside this codebase - but its cleaner to have it track its own state.

Reply:
okay cool, sure. would you mind making the log message a little more detailed? maybe something along the lines of like `Websocket did not stop successfully as the feed has not been started yet." also feels like it should be a warning as the user is doing something incorrectly.... then lets merge it

Reply:
OK, changed message to:
Websocket did not attempt to stop as the feed has not been started yet

and changed it to a warning.

Yeah - I get this happen if something else odd external has happened, so the app is shutting down before its got going. Things like network issue / new install has wrong configuration / bad db connection string etc.